style.css

```diff
@@ -743,3 +747,677 @@ button:active {
+    width: 100%;
+    height: 924px;
+    background: url("images/abt-myself-desktop.png") no-repeat;
+    border-top-right-radius: 80px;
+    top: 3600px;
+  }
+
+  .top {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 0;
+    gap: 24px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    width: 447px;
+    height: 392px;
+    margin-left: 142px;
+    margin-top: 142px;
+  }
+
+  h2.about-title {
+    width: 447px;
+    height: 114px;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    color: #091e42;
+    margin-top: 0;
+  }
+
+  p.hello {
+    width: 447px;
+    height: 102px;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  .Social-media-layout .social-media-icons {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    width: 200px;
+    height: 40px;
+    margin: 0;
+    padding: 0;
+  }
+
+  .skills-list {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 0;
+    width: 684px;
+    height: 360px;
+    margin-top: 142px;
+    margin-right: 142px;
+  }
+
+  .frame-languages {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 0;
+    width: 684px;
+    height: 216px;
+  }
+
+  .languages {
+    width: 684px;
+    height: 72px;
+  }
+
+  .lang {
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 24px;
+    display: flex;
+    align-items: center;
+    color: #344563;
+  }
+
+  .arrow {
+    left: 25%;
+    right: 25%;
+    top: 37.5%;
+    bottom: 37.5%;
+  }
+
+  .frame2 {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 12px;
+    gap: 12px;
+    width: 414px;
+    height: 144px;
+  }
+
+  .list {
+    width: 122px;
+    height: 120px;
+    background: #f7f7f9;
+    border-radius: 8px;
+  }
+
+  .language {
+    width: 98px;
+    height: 24px;
+    margin-left: 12px;
+    margin-top: 76px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    display: flex;
+    align-items: center;
+    color: #253858;
+  }
+
+  .frameworks,
+  .skills {
+    width: 684px;
+    height: 72px;
+  }
+
+  hr.solid {
+    text-align: center;
+    border-top: 3px solid #dfe1e6;
+    width: 100%;
+    margin-left: 0;
+  }
+
+  .footer {
+    top: 4515px;
+    position: absolute;
+    left: 0;
+  }
+
+  .contact {
+    background: #6070ff;
+    box-sizing: border-box;
+    padding: 103px 24px 8px 24px;
+    border-radius: 76px 0 0 0;
+    background-image: url("./Images/contactformbackgroundshapesdesktop.png");
+    background-repeat: no-repeat;
+    background-size: contain;
+    display: flex;
+    flex-direction: column;
+    width: 99vmax;
+    height: 846px;
+    align-items: center;
+    margin: 0 auto;
+    text-align: center;
+    justify-content: center;
+  }
```

Comment:
- Please, kindly ensure the background image for the desktop version is showing. please fix this issue.

Reply:
It is now showing. thank you very much